src/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-var assign = require('lodash.assign');
+let assign = require('lodash.assign');
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).